data/blog/2024-year-in-review.mdx

```diff
@@ -0,0 +1,545 @@
+date: '2024-12-28'
+tags: ['OpenSats', 'grants', 'donations', 'bitcoin', 'nostr', 'review']
+draft: false
+authors: ['dergigi']
+images: ['/static/images/blog/66-opensats-2024-year-in-review.jpg']
+summary: "365 days. Two funds. 200+ grants. Billions of sats."
+---
+
+Twenty-twenty-four. What a year! A lot has happened in the past 12 months, and
+just like last year, we want to take the opportunity to pause, reflect, and
+share our year in review. We want to look back at the announcements we made,
+share an update in regards to the donations we received, and give an overview of
+how we allocated our funds. After that we will share updates to our grantee
+statistics and provide an outlook of what the next year might bring.
+
+- [Announcements](#announcements)
+- [Donations](#donations)
+- [Allocation](#allocation)
+- [Grantees](#grantees)
+- [Outlook](#outlook)
+
+But before we dive into the various graphs and numbers, let us spend a minute to
+remind everyone—and most importantly ourselves—_why_ we are doing what we are
+doing.  _"I'm convinced that to give away a dollar effectively is harder than to
+make a dollar,"_ Steve Jobs once mentioned in an interview. And indeed, making
+sure that grants are given to the right projects at the right time—in other
+words: making sure that [grant selection](/selection) is done fairly and
+properly—is an incredibly hard problem.
+
+Why set out to solve this problem in the first place? Why go through the trouble
+of creating a non-profit entity? Why spend countless hours reviewing grant
+applications, commits, reports, pull requests, and other software artifacts? Why
+bother with open-source grants at all? Shouldn't this problem be solved by
+companies, or direct crowd funding, or the state?
+
+From the beginning, the vision of OpenSats was to create a sustainable
+foundation that provides [consistent funding](/about) for free and open-source
+projects, especially those who make bitcoin flourish. We don't want to be
+dependent on quarterly profits nor on funding from the state, which is why we
+chose a public charity as a vehicle.
+
+As mentioned in our [mission statement](/mission), we believe that we can help
+make bitcoin more accessible, robust, secure, and decentralized. Further, we
+believe that the projects we support have the potential to revolutionize the way
+we think about individual sovereignty and liberty—values that we hold in high
+regard. We want to operate with as little overhead as possible, which is why we
+don't take a cut from donations to our funds. And we don't want any one person
+or small group to have control over funding or organizational decisions, which
+is why we have a 9-person board that is the final arbiter of all decisions.
+
+We don't claim to have all the answers, and we don't claim that what we are
+doing is frictionless or perfect. We are bound to make mistakes, and we will
+always be prone to false positives (funding projects that shouldn't receive
```

Comment:
probably can leave out the comment about "false positives (funding projects that shouldn't receive....", that sounds bad

Reply:
I'll rephrase that.